x-pack/plugins/security_solution/public/resolver/store/data/selectors.ts

```diff
@@ -167,6 +168,105 @@ export function hasMoreAncestors(state: DataState): boolean {
   return tree ? resolverTreeModel.hasMoreAncestors(tree) : false;
 }
 
+/**
+ * A map of `entity_id`s to functions that provide information about
+ * related events by ECS `.category` Primarily to avoid having business logic
+ * in UI components.
+ */
+export const relatedEventInfoByEntityId = createSelector(
```

Comment:
would you please explicitly type the return value of this selector.

Reply:
https://github.com/elastic/kibana/pull/70097/commits/7ce05719032182d0ccecf8524a37856b114cf4ef